extension/json/src/reader/buffered_json_reader.cpp

```diff
@@ -49,12 +64,12 @@ std::unique_ptr<storage::MemoryBuffer> BufferedJsonReader::removeBuffer(
     return result;
 }
 
-void BufferedJsonReader::throwParseError(yyjson_read_err& err, const std::string& extra) const {
-    auto unit = options.format == JsonScanFormat::NEWLINE_DELIMITED ? "line" : "record/value";
-    // TODO(Ziyi): report error line number.
-    throw common::RuntimeException{
-        common::stringFormat("Malformed JSON in file \"{}\", at byte {} in {}: {}. {}", fileName,
-            err.pos + 1, unit, err.msg, extra)};
```

Comment:
I removed the byte offset from the error message since I think having the record + reconstructed record should be sufficient